### PR TITLE
diode experiment working properly

### PR DESCRIPTION
**Pull request by NamanRawat1**
Fixes issue #654 

APK-
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1366418/app-debug.zip)

Screenshots for the change: 
screenshot_2017-10-08-02-08-27-599_org fossasia pslab

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/fragment/ControlFragmentMain.java [hunk 1]
```diff
@@ -25,6 +26,17 @@ public static ControlFragmentMain newInstance() {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+    }
```
[the remark this answers is not in the document]
Comment (NamanRawat1) [reply app/src/main/java/org/fossasia/pslab/fragment/ControlFragmentMain.java]: oh it comes by mistake i will update it soon. @asitava1998

**Comment by NamanRawat1** [comment]
apk updated-
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1385349/app-debug.zip)

**Comment by viveksb007** [comment]
@NamanRawat1  Please pull changes from branch and push another commit to fix the build.

**Comment by NamanRawat1** [comment]
Why is it showing so many files changed?  Had I done something wrong? @viveksb007

**Comment by viveksb007** [comment]
Looks like you messed up with commit history. As a result its showing previous changes in your PR. You can send a new PR containing your changes only.

**Comment by NamanRawat1** [comment]
Sent the new PR to review  @viveksb007

**Pull request closed without merging**